### DISCUSSION TITLE
importing obj error~	                                           );

### DISCUSSION THREAD

**Discussion by summerjet** [assimp/assimp#5742] [Q&A]
when importing obj file using function: importer.ReadFile(filename, aiProcess_Triangulate),	mNumFaces=58, mNumVertices=120,i don't know what's going on!
![screenshot-20240831-133345](https://github.com/user-attachments/assets/c82ad5cc-951f-4254-af87-a1d22a65970a)

**Comment by kimkulling** [comment]
What do you want to do? Seems that the import was right. If you need some help rendering the model, google for it or look into our samples. There you should be able to find some help.
